src/com.pentalog.pentastagiu/week3/Bank.java

```diff
@@ -0,0 +1,60 @@
+package com.pentalog.pentastagiu.week3;
+
+public class Bank {
+    Scanner input = new Scanner(System.in);
+            System.out.println("Enter your Name: ");
+    String customerName = input.nextLine();
+            System.out.println("Enter Account Type: ");
+    String type = input.next();
+            System.out.println("Enter Initial Balance: ");
+
+    int balance = input.nextInt();
+      Bank balance1 = new BankAccount(customerName, amount, type, balance);
```

Comment:
amount nu e declarat deasupra acestei linii, deci asta e un motiv in plus pentru care nu compileaza, nu stie cine e amount